packages/camelsnakekebab/camelsnakekebab.0.3/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+synopsis: "A Ocaml library for word case conversion"
+description:"""
+conversion bettween upper_camel_case, lower_camel_case 
+lower_snake_case, lower_snake_case, upper_snake_case,
+constant_case, kebab_case, http_header_case 
+"""
+maintainer: "Hao Wu <echowuhao@gmail.com>"
+authors: ["Hao Wu <echowuhao@gmail.com>"]
+license:  "MIT"
+build: ["dune" "build" "-p" name]
+homepage: "https://github.com/swuecho/camelsnakekebab"
+bug-reports: "https://github.com/swuecho/camelsnakekebab/issues"
+dev-repo: "git+https://github.com/swuecho/camelsnakekebab.git"
+depends: [
+  "ocaml" {>= "4.04.1"}
+  "dune" {build}
+  "base"
+  "pcre"
+  "ounit" {with-test}
```

Comment:
You have test dependencies but no run-test section or {with-test} section in build

Reply:
oh, that sounds like something that could/should be added to `opam lint`?